ocaml/xapi/xapi_pool_update.ml

```diff
@@ -453,9 +458,11 @@ let destroy ~__context ~self =
 let detach_attached_updates __context =
   Db.Pool_update.get_all ~__context |>
   List.iter ( fun self ->
-      Helpers.log_exn_continue ("detach_attached_updates: update_uuid " ^ (Db.Pool_update.get_uuid ~__context ~self))
-        (fun () -> ignore(Helpers.call_api_functions ~__context
-                            (fun rpc session_id -> Client.Pool_update.detach ~rpc ~session_id ~self))) ()
+      let uuid = Db.Pool_update.get_uuid ~__context ~self in
```

Comment:
We request `uuid` and `vdi` here, but in `detach` function, we request these two again. I am wondering if we can improve them by not requesting Db action too much.

Reply:
Updated.

Reply:
👍